src/dbup-sqlserver/SqlServerExtensions.cs

```diff
@@ -191,10 +250,12 @@ public static void SqlDatabase(this SupportedDatabasesForEnsureDatabase supporte
                 }
             }
 
+            string collationString = String.IsNullOrEmpty(collation) ? "" : string.Format(@" COLLATE {0}", collation);
```

Comment:
Minor nitpick to be consistent with referencing `string`:

```suggestion
            string collationString = string.IsNullOrEmpty(collation) ? "" : string.Format(@" COLLATE {0}", collation);
```